src/intl/en/page-what-is-ethereum.json

```
@@ -3,52 +3,49 @@
   "page-what-is-ethereum-meet-comm": "Meet the community",
   "page-what-is-ethereum-meta-description": "Learn about Ethereum, what it does and how to try it for yourself.",
   "page-what-is-ethereum-meta-title": "What is Ethereum?",
   "page-what-is-ethereum-p2p-card": "A peer-to-peer network",
-  "page-what-is-ethereum-p2p-card-desc": "Ethereum allows you to move money, or make agreements, directly with someone else. You don't need to go through intermediary companies.",
+  "page-what-is-ethereum-p2p-card-desc": "Ethereum allows you to coordinate, make agreements or transfer digital assets directly with other people. You don't need to rely on intermediaries.",
   "page-what-is-ethereum-start-building-btn": "Start building",
   "page-what-is-ethereum-title": "What is Ethereum?",
   "page-what-is-ethereum-subtitle": "A complete beginner's guide to how Ethereum works, the benefits it brings and how it's being used by millions of people around the world.",
   "page-what-is-ethereum-button-lets-start": "Let's start",
   "page-what-is-ethereum-blockchain-tab-title": "What is a blockchain?",
   "page-what-is-ethereum-blockchain-tab-content": "<b>A blockchain is a database of transactions</b> that is updated and shared across many computers in a network. Every time a new set of transactions is added, its called a “block” - hence the name blockchain. Most blockchains are public, and you can only add data, not remove. If someone wanted to alter any of the information or cheat the system, they’d need to do so on the majority of computers on the network. That is a lot! This makes established blockchains like Ethereum highly secure.",
-  "page-what-is-ethereum-cryptocurrency-tab-title": "Why is it called cryptocurrency?",
-  "page-what-is-ethereum-cryptocurrency-tab-content": "Blockchains use cryptographic techniques to ensure that your funds are safe. Similar techniques have been used in the banking industries to ensure the security of monetary transactions for years. So you could say cryptocurrencies have a bank level of security.",
+  "page-what-is-ethereum-cryptocurrency-tab-title": "What is a cryptocurrency?",
+  "page-what-is-ethereum-cryptocurrency-tab-content-1": "Cryptocurrency is a term used to describe many types of fungible digital tokens secured using a blockchain. It all started with Bitcoin. Bitcoin can be used to transfer value between two parties without having to trust a middleman. You only have to trust the Bitcoin code, which is all open and freely available.",
+  "page-what-is-ethereum-cryptocurrency-tab-content-2": "The reason assets such as Bitcoin and Ether are called “cryptocurrencies” is that the security of your data and assets is guaranteed by cryptography, not by trusting an institution or corporation to act honestly.",
```

Small capitalization check... typically we use "Bitcoin" for protocol/network, "bitcoin"/"ether" for the currency. The above uses look fine, but I'd suggest lowercasing these... hopefully doesn't add confusion.

```suggestion
  "page-what-is-ethereum-cryptocurrency-tab-content-2": "The reason assets such as bitcoin and ether are called “cryptocurrencies” is that the security of your data and assets is guaranteed by cryptography, not by trusting an institution or corporation to act honestly.",
```